src/operators.jl

```diff
@@ -272,6 +272,8 @@ end
 
 *{T<:QuadExpr,S<:Union(Variable,AffExpr,QuadExpr)}(::T,::S) =
     error( "*(::$T,::$S) is not defined. $op_hint")
+(*)(lhs::GenericQuadExpr{Float64,Variable}, rhs::GenericQuadExpr{Float64,Variable}) =
```

Comment:
You can use the ``QuadExpr`` alias instead of ``GenericQuadExpr{Float64,Variable}``